lightning/src/ln/channelmanager.rs

```diff
@@ -13293,9 +13303,7 @@ where
 					if let Some(short_channel_id) = channel.context.get_short_channel_id() {
 						short_to_chan_info.insert(short_channel_id, (channel.context.get_counterparty_node_id(), channel.context.channel_id()));
 					}
-					if let Some(funding_txo) = channel.context.get_funding_txo() {
-						outpoint_to_peer.insert(funding_txo, channel.context.get_counterparty_node_id());
-					}
+					outpoint_to_peer.insert(funding_txo, channel.context.get_counterparty_node_id());
```

Comment:
Just confirming this should have always been `Some` before given the check on line 13222.